packages/cli-platform-android/src/commands/buildAndroid/index.ts

```diff
@@ -0,0 +1,54 @@
+import {CLIError, logger} from '@react-native-community/cli-tools';
+import {Config} from '@react-native-community/cli-types';
+import execa from 'execa';
+import {getAndroidProject} from '../../config/getAndroidProject';
+import {toPascalCase} from '../runAndroid/runOnAllDevices';
+
+export interface BuildFlags {
+  mode: 'debug' | 'release';
+  '--variant'?: string;
+}
+type AndroidProject = NonNullable<Config['project']['android']>;
+
+async function buildAndroid(
```

Comment:
Would it make sense to have this function exported or shared, so it could be reused from the `run-android`?